lib/src/parse/stylesheet.dart

```diff
@@ -57,6 +57,20 @@ abstract class StylesheetParser extends Parser {
   /// Whether the parser is currently within a parenthesized expression.
   var _inParentheses = false;
 
+  SilentComment _lastSilentComment;
+
+  set lastSilentComment(SilentComment comment) => _lastSilentComment = comment;
+
+  /// Returns the most recent silent comment encountered by this parser.
+  ///
+  /// The value is read-once and will be null after a read until the next
+  /// comment is encountered.
+  SilentComment clearLastSilentComment() {
```

Comment:
Similarly, I think having this is just adding more complexity than is necessary. I don't think it's worth the extra API complexity to enforce the expected usage pattern of this field, especially when this class at least could just access the private field anyway.

Reply:
OK, I can expose the field, but are you asking that I remove this method all together and just set the field to null at the same location every time it's read?

Reply:
Done.